### PR TITLE
Add more configuration items for resource server

### PR DESCRIPTION
**Pull request by chenrujun**
Add more configuration items for resource server

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @moarychan , @yiliuTo , @zhichengliu12581 , please help to review this PR.

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADResourceServerProperties.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import org.springframework.util.StringUtils;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Configuration properties for Azure Active Directory B2C.
+ */
+@ConfigurationProperties("azure.activedirectory.resource-server")
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADResourceServerProperties.java:19]: @yiliuTo , maybe should should use the new property prefix? Or support both new and old property prefix at the same time.

**Comment by moarychan** [comment]
Hi @chenrujun , is it possible to apply this feature to the b2c starter at the same time?

**Comment by chenrujun** [comment]
@moarychan , 

>  is it possible to apply this feature to the b2c starter at the same time?

Good point, I'll do it in another PR.

**Comment by chenrujun** [comment]
Hi, @moarychan , @yiliuTo , @zhichengliu12581 .
Please help to review this PR.
And comment `LGTM` if you think it's OK to merge.

**Comment by moarychan** [comment]
LGYM.

**Comment by zhichengliu12581** [comment]
LGTM

**Comment by chenrujun** [comment]
After discuss with @moarychan , we think that we do not need such kind of property, because current `AADB2CJwtBearerTokenAuthenticationConverter` only used in test code and sample code.

**Pull request merged**